iota-client/src/client.rs

```diff
@@ -646,7 +646,7 @@ impl Client {
 
     /// Return the balance in iota for the given addresses; No seed or security level needed to do this
     /// since we are only checking and already know the addresses.
-    pub async fn get_address_balances(&self, addresses: &[Address]) -> Result<Vec<AddressBalancePair>> {
+    pub async fn get_address_balances(&self, addresses: &[String]) -> Result<Vec<AddressBalancePair>> {
```

Comment:
It might be good to define the address type so as to enhance the readability and maintainability. How do you think?
```rust
type Address = String;
```

Reply:
Awesome, thanks for the suggestion